Documentation for github app installation (#19)
simplybusiness/dobby: README.md

## Bump version

### Installation

> Dobby require a github app to be installed on your repository or organization wide. If you have already created a github app you can follow instruction from step 3 otherwise follow these steps: 

Reviewer: ```suggestion
Dobby requires a Github App to be installed either on an individual repository or organization wide. If you have already created a Github App you can follow instruction from step 3. Otherwise follow these steps: 
```

1. [Create a minimal GitHub App](https://docs.github.com/en/developers/apps/creating-a-github-app), setting the following fields:
   - Set GitHub App name. 
   - Set Homepage URL to your github repository.
   - Uncheck Active under Webhook. You do not need to enter a Webhook URL.
   - Under Repository permissions: Contents select Access: Read & write.
   - Under Repository permissions: Pull requests select Access: Read & write. 

2. Create a Private key from the App settings page and store it securely.

> 3. Install the App on your repository/organization wide.

Reviewer: ```suggestion
3. Install the App either on your repository or organization wide.
```

> 4. Set secrets on your repository or on organization containing the GitHub App ID, and the private key you created in step 2 as DOBBY_APP_ID, DOBBY_PRIVATE_KEY

Reviewer: ```suggestion
4. Set secrets on your repository or organization containing the GitHub App ID, and the private key you created in step 2 as DOBBY_APP_ID, DOBBY_PRIVATE_KEY
```

Reviewer: Is the private key used for both fields? Or is `DOBBY_APP_ID` something else?

Author: `DOBBY_APP_ID ` is the github app ID.
